frontend/src/ModelOverview.tsx

```diff
@@ -69,21 +71,51 @@ function ModelOverview({ version }: ModelOverviewProps) {
             </Box>
           </Box>
           <Box sx={{ p: 2 }}>
-            <Typography variant='h6'>{schema?.schema.properties.contacts.properties.uploader.title}</Typography>
+            <Stack spacing={0.5} direction='row'>
```

Comment:
You can horizontally align the text and icon by using `alignItems='center'` on the `Stack`
```
<Stack spacing={0.5} direction='row' alignItems='center'>
```